lib/Dancer2/Manual.pod

```diff
@@ -178,16 +178,12 @@ requests.
 =head2 Route Handlers
 
 The route action is the code reference declared. It can access parameters
-through the C<params> keyword, which returns a hashref. This hashref is a
+through the specific C<route_parameters>, C<query_parameters>, and C<body_parameters> keywords, which return an L<Hash::MultiValue> object. This hashref is a
```

Comment:
No worries, will push a commit to fix. I originally had them as `L<hash>` but for some reason got in my head the `L<Hash::MultiValue>` was correct.